rpm_spec/subpackages/manageiq-system

```diff
@@ -38,3 +38,4 @@ Requires: openldap-clients
 %{_sysconfdir}/manageiq/postgresql.conf.d/01_miq_overrides.conf
 %{_sysconfdir}/profile.d/evm.sh
 %{_sysconfdir}/sudoers.d/repmgr
+%exclude %{_sysconfdir}/default/manageiq-appliance.properties
```

Comment:
Not sure if order matters, but can you move this up 4-ish lines to be under the `%{_sysconfdir}/default/manage*.properties` ?

Reply:
I thought it looked weird to split up the `%{_sysconfdir}` group with an exclude in the middle